website_checker.py

```diff
@@ -68,7 +68,10 @@ def check_now(a,urls):
 	Code to check if a URL has updated
 	'''
 	# For KL
-	today = datetime.now(pytz.timezone('Asia/Calcutta')).strftime('%d/%m/%Y')
+	today_1 = datetime.now(pytz.timezone('Asia/Calcutta')).strftime('%DD/%MM/%YYYY')
```

Comment:
I dont think this is valid format for strftime.` %d/%m/%Y` would give the date as 05/08/2020